src/queues/UnlistedUserRedactionQueue.ts

```diff
@@ -13,12 +13,19 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
 import { extractRequestError, LogLevel, LogService, MatrixClient, Permalinks } from "matrix-bot-sdk";
 import { logMessage } from "../LogProxy";
 import config from "../config";
 
-export class AutomaticRedactionQueue {
+/**
+ * This class is a queue of users who have been flagged
+ * for redaction by the flooding or image protection.
```

Comment:
Nit: `typically by the flooding or image protection` (more reasons could be added later).